### PR TITLE
Editor box plugin: frontend

### PR DESCRIPTION
**Pull request by elbotho**
[226222](https://frontend-git-editor-box-plugin-frontend-serlo.vercel.app/226222)

### CODE REVIEW

**Comment by Entkenntnis** [comment]
bundlesize again -_-

increased to 180kb, some of the imports is pulling in a lot of dependencies into the [...slug]

File: src/edtr-io/plugins/box/renderer.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { useInstanceData } from '@/contexts/instance-context'
 import { useLoggedInData } from '@/contexts/logged-in-data-context'
 import { hasOwnPropertyTs } from '@/helper/has-own-property-ts'
 
-const boxTypeStyle = {
+export const boxTypeStyle = {
   blank: {},
   example: {},
   quote: { icon: faQuoteRight },
```
Comment (Entkenntnis) [inline src/edtr-io/plugins/box/renderer.tsx]: maybe moving this out of this file into the frontend will do the trick (and import from there)

Comment (elbotho) [reply src/edtr-io/plugins/box/renderer.tsx]: ah damn, was afraid that might happen. we really have to take care when we try to split editor plugins into a render and editor part in the future.

**Comment by elbotho** [comment]
okay, we are at `+2kb` compared to staging now.

**Comment by Entkenntnis** [comment]
thank you 👍

**Pull request merged**